### DISCUSSION TITLE
How to fix label spacing and printing blanks?

### DISCUSSION THREAD

**Discussion by lonelyadmin** [snipe/snipe-it#10199] [Q&A]
I left a gap under the text fields on the label to accommodate possible multi-line serial numbers or model numbers that wrap. However, when they do it pushes down the barcode. How can I keep this from happening?

Also, when printing labels to a Dymo 450 (Generate -> print from Chrome) there is always a blank label, I have to manually choose print current page (or every other). How do I fix this
![bad](https://user-images.githubusercontent.com/33500076/137386693-f905129c-8091-4820-bd4e-8dea9acbb42f.png)
![good](https://user-images.githubusercontent.com/33500076/137386694-fbbda03e-2de7-4f5c-af07-773ace18e9bd.png)
?

**Comment by MchalesNavy** [comment]
The barcode is being pushed down because the text wrapping is creating a new line, not wrapping onto the next line.  I'm not sure it can actually work as you're describing.  The space you've left below is allocated to other fields such as asset name etc.  I don't know for certain, but the only way I know to squeeze a larger serial onto the label like that is to reduce the text size.

As for the blanks you're getting, that's probably due to the page size not matching your label size.  Sounds simple but you're in for an adventure.  Go to Settings > Labels.  Make sure the page size is set to the size of your label.  You're then going to need to mess around with margins and spacing etc.  If there's an easy way to do this I don't know what it is, trial and error.  Basically though if the chrome preview shows you more than one page, you haven't got the settings right yet.  You might also need to mess with the page margins in the chrome print dialogue.  Print - more settings - margins.  Mine is set to custom with 0mm top and bottom, 3mm to the left and 5.5mm to the right.  If I try to widen it on the right it creates a blank page.  Why?  Don't know.

Good luck.
